### ISSUE TITLE
Incorrect version in 6.74.0 (it's 6.73.0)

### ISSUE THREAD

**Issue by mgorny** [HypothesisWorks/hypothesis#3627]
The 6.74.0 tag has a version of 6.73.0 embedded:

https://github.com/HypothesisWorks/hypothesis/blob/hypothesis-python-6.74.0/hypothesis-python/src/hypothesis/version.py#L11

This causes test failures, e.g.:

```pytb
_____________________________________________ test_does_not_run_reporting_hook_by_default _____________________________________________
[gw1] linux -- Python 3.9.16 /tmp/portage/dev-python/hypothesis-6.74.0/work/hypothesis-hypothesis-python-6.74.0/hypothesis-python-pypy3/install/usr/bin/pypy3
Traceback (most recent call last):
  File "/tmp/portage/dev-python/hypothesis-6.74.0/work/hypothesis-hypothesis-python-6.74.0/hypothesis-python/tests/pytest/test_profiles.py", line 41, in test_does_not_run_reporting_hook_by_default
    assert __version__ in out
AssertionError: assert '6.73.0' in '========================================================= test session starts =========================================================\nplatform linux -- Python 3.9.16[pypy-7.3.11-final], pytest-7.3.1, pluggy-1.0.0\nrootdir: /tmp/portage/dev-python/hypothesis-6.74.0/temp/pytest-of-portage/pytest-0/popen-gw1/test_does_not_run_reporting_hook_by_default0\ncollected 1 item\n\ntest_does_not_run_reporting_hook_by_default.py .                                                                                [100%]\n\n========================================================== 1 passed in 0.02s =========================================================='
-------------------------------------------------------- Captured stdout call ---------------------------------------------------------
running: /tmp/portage/dev-python/hypothesis-6.74.0/work/hypothesis-hypothesis-python-6.74.0/hypothesis-python-pypy3/install/usr/bin/pypy3 -mpytest --basetemp=/tmp/portage/dev-python/hypothesis-6.74.0/temp/pytest-of-portage/pytest-0/popen-gw1/test_does_not_run_reporting_hook_by_default0/runpytest-0 /tmp/portage/dev-python/hypothesis-6.74.0/temp/pytest-of-portage/pytest-0/popen-gw1/test_does_not_run_reporting_hook_by_default0/test_does_not_run_reporting_hook_by_default.py --hypothesis-profile test --basetemp=/tmp/portage/dev-python/hypothesis-6.74.0/temp/pytest-of-portage/pytest-0/popen-gw1/basetemp
     in: /tmp/portage/dev-python/hypothesis-6.74.0/temp/pytest-of-portage/pytest-0/popen-gw1/test_does_not_run_reporting_hook_by_default0
========================================================= test session starts =========================================================
platform linux -- Python 3.9.16[pypy-7.3.11-final], pytest-7.3.1, pluggy-1.0.0
rootdir: /tmp/portage/dev-python/hypothesis-6.74.0/temp/pytest-of-portage/pytest-0/popen-gw1/test_does_not_run_reporting_hook_by_default0
collected 1 item

test_does_not_run_reporting_hook_by_default.py .                                                                                [100%]

========================================================== 1 passed in 0.02s ==========================================================
```

**Comment by Zac-HD** [comment]
Ugh, yeah, we've had some broken releases due to a change in the GitHub API on... Tuesday?  Should be fixed by #3626 😕

**Comment by mgorny** [comment]
Thank you!
